index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="./style/styled.css">
+        <script src="https://kit.fontawesome.com/0348c0adc9.js" crossorigin="anonymous"></script>
+        <title>Mocking Mint.com</title>
+    </head>
+    <body id="wrapper">
+        <header>
+            <div class="signin-container">
+                <div class="signin-text">
+                        Already have an Intuit account?<br>
+                    <a href="#">
+                        I forgot my User ID or Password 
+                    </a>
+                </div>
+                <div class="submit">
+                    <button type="button" class="button-signin">
+                        <i class="fas fa-lock"></i>
+                            Sign In
+                    </button>
+                </div>
+            </div>
+            <div class="header-icons">
+                <img class="logo" alt="Intuit" src="./style/img/logo.png"><br>
+                <div class="icons">
+                    <img class="mint" alt="Mint" src="./style/img/mint.png">
+                    <img class="quickbooks" alt="Quickbooks" src="./style/img/quickbooks.png">
+                    <img class="turbotax" alt="Turbotax" src="./style/img/turbotax.png">
+                </div>
+            </div>
+        </header>   
+        <main>
+            <div class="form-container">
+            <h2>
+                Create an Intuit account
+            </h2>
+            <div  class="main-text">
+            <p>
+                One account for everything Intuit, including Mint.
+            </p>
+            <p>
+                <a href="#">
+                    Learn more
+                </a>
+            </p>
+            </div>
+            <form>
+                <label for="email">Email address</label><br>
+                <input type="email" id="email" name="email" required><br>
+                <label for="phone">Phone (recommended)</label><br>
+                <input type="number" id="phone" name="phone" required><br>
```

Comment:
- [ ] Since you are expecting the user to input a phone number, you are expected to use the `type="tel"` for that input.